third_party/amd/lib/TritonAMDGPUTransforms/StreamPipeline.cpp

```diff
@@ -129,9 +129,23 @@ class StreamPipeliner {
     options.supportDynamicLoops = true;
     options.peelEpilogue = true;
     options.predicateFn = streamPredication;
+
+    // Annotate loadOp in prologue for further moving up
+    options.annotateFn = [this](Operation *op,
+                                tt::PipeliningOption::PipelinerPart part,
+                                unsigned stage) {
+      if (part != tt::PipeliningOption::PipelinerPart::Prologue)
+        return;
+
+      if (auto loadOp = dyn_cast<tt::LoadOp>(op)) {
+        loadOp->setAttr("PipelinerPart",
```

Comment:
Maybe something like `amdgpu.pipeliner_part` to have a prefix to avoid potential collision.